README.md

```diff
@@ -52,10 +56,55 @@ https://user-images.githubusercontent.com/7124846/235194069-e2d65111-1440-4f85-a
 
 **Web:**
 
-1. Navigate to the frontend web project: `create_link/frontend_web`
+1. In a separate shell, change directories to the frontend web project:
+
+   ```sh
+   cd create_link/frontend_web
+   ```
+
 2. Install dependencies and start the server.
    ```sh
    npm install
    npm run dev
    ```
-3. Visit `http://localhost:3000` in your browser and create a new link.
+
+3. At this point, the frontend should be running on port 3000 (in npm) and you should see "Local:  **http://localhost:3000**".
+
+4. Click that to visit `http://localhost:3000` in your browser, and create a new link.
+
+## Get some account data!
+
+If you've followed the steps above, created a new link, and clicked Next all the way to the end, you should have been rewarded with a panel that shows a new `link_id`:
+
+   > **link_id**<br>
+   > mk_AfZGwnseamMJQcAWXqk7iL
+
+You can now use Postman, curl, or any HTTP client to request accounts, balances, and other data about that link
+using the MoneyKit API:  https://docs.moneykit.com.
+
+   ```sh
+   # create an access_token
+   curl --request POST \
+   --url 'https://production.moneykit.com/auth/token'
+   --header 'accept: application/json'
+   --header 'content-type: application/x-www-form-urlencoded'
+   --data 'grant_type: client_credentials'
+   --data 'client_id: your_client_id'
+   --data 'client_secret: your_secret'
+
+   # get accounts
+   curl --request GET
+   --url 'https://production.moneykit.com/links/{link_id}/accounts'
+   --header 'accept: application/json'
+   --header 'Authorization: Bearer { your_auth_token }'
```

Comment:
While I would hope the reader would figure out themselves that the braces here are just marking out the placeholder, I think this might be clearer without them, in case they are very unfamiliar with header syntax or curl etc.

```suggestion
   --header 'Authorization: Bearer YOUR_AUTH_TOKEN'
```

We also reference an "auth" token here vs. an "access" token in the comment above the first request above, we probably want to make those consistent.